src/cumulus/commands/init.py

```diff
@@ -13,49 +13,53 @@ def init_docker_compose(self):
             'version': DOCKER_COMPOSE_VERSION,
             'services': {}
         }
-
+        project = "example"
+        hostname = project+".cumulus.com"
         self.already_started = []
         new_services = self.options['<service>']
         clean = self.options['--clean']
         for service in new_services:
             if(os.path.exists('./'+service) and not clean):
                 self.already_started.append(service)
                 continue
-                
+            name = project+"_"+service
             if service in WEB_APP:
                 doc['services'][service] = {
-                    'image': DOCKER_HUB + service,
+                    # 'image': DOCKER_HUB + service, uncomment before merging
+                    'container_name': name,
+                    'build': {'context': './service_images/django'},
                     'volumes': [os.getcwd()+':/cumulus'],
-                    'ports': ['8080:8080']
-                }
+                    'ports': [PORTS[service]+':'+PORTS[service]],
+                    'command': COMMANDS[service],
+                    'network_mode': 'bridge'}
```

Comment:
why network_mode bridge?

Reply:
Yea good catch, I was testing out the DNS shit.